installscripts/jazz-terraform-unix-noinstances/scripts/configure-gitlab.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Just FYI and for future reference this can be done in about 10 lines of python with the python gitlab library, but I've already done that in the python installer rewrite branch (`configure-gitlab.py`) so no big deal right for now.